invenio_theme_tugraz/templates/invenio_theme_tugraz/accounts/login_user.html

```diff
@@ -25,7 +25,7 @@ <h3 class="ui login header">{{ _('Log in to Repository') }}</h3>
         {%- endblock form_header %}
         <div class="ui divider"></div>
         <!--Log in with SSO-->
-        {% if config.OAUTHCLIENT_REMOTE_APPS or config.SSO_SAML_IDPS %}
+        {% if config.OAUTHCLIENT_REMOTE_APPS or config.SSO_SAML_IDPS or config.EDUGAIN_LOGIN_ENABLED %}
```

Comment:
the addition could cause problems on the next div tag or? because `OAUTHCLIENT_REMOTE_APPS` could be false but the for loop needs the value and would fail then?

Reply:
this is already an issue with just the 2 variables right now:
if one is defined and the other isn't, this `if` passes and the template then chokes below on the undefined one

might as well fix it while I'm at it
Fixed